services/121-service/src/notifications/sms/sms.controller.ts

```diff
@@ -10,6 +12,22 @@ export class SmsController {
     this.smsService = smsService;
   }
 
+  @Roles(UserRole.Admin)
+  @ApiResponse({
+    status: 200,
+    description: 'Test controller to test sending sms',
+  })
+  @ApiImplicitParam({ name: 'number' })
+  @Get(':number')
+  public async sendSms(@Param() params): Promise<void> {
+    return await this.smsService.notifyBySms(
+      params.number,
+      'en',
+      'included',
+      1,
```

Comment:
@RubenGeo Are there any 'named' variables we could use here? Like a "InclusionStatus"-enum?

It 'feels' a bit brittle to just use these string-values here, and the 'overhead' of the extra `.enum.ts`-file seems minimal...